demo/create_imdb.sql

```diff
@@ -0,0 +1,19 @@
+DROP TABLE IF EXISTS imdb_title_basics;
+CREATE TABLE imdb_title_basics (
+  tconst TEXT,
+  "titleType" TEXT,
```

Comment:
Unfortunately this is necessary because duckdb doesn't respect any quotes when it creates table in postgres.